RebrickableSharp.Client.Csv/RebrickableCsvLoader.cs

```diff
@@ -0,0 +1,199 @@
+﻿using CsvHelper;
+using CsvHelper.Configuration;
+using System.Globalization;
+using System.IO.Compression;
+
+namespace RebrickableSharp.Client.Csv;
+
+/// <summary>
+/// <para>
+/// A class to download and parse Csv static files from the Rebrickable download CDN.
+/// </para>
+/// <para>See <see cref="https://rebrickable.com/downloads/"/></para>
+/// </summary>
+public class RebrickableCsvLoader : IRebrickableCsvLoader

```

Comment:
Should we accept an (optional) CancelationToken for all async methods in this class?

Reply:
I guess we could yes, I'll take a look.

On a similar note, not sure that we would need to add more as the MeasureRequestAsync of the regular client. The http calls from the CsvLoader are fairly straight forward. What do you think?

Reply:
[096c548](https://github.com/gebirgslok/RebrickableSharp/pull/6/commits/096c548e49a8e4dde9758a54b3add9d3b4277160)